test/show.jl

```diff
@@ -199,18 +199,25 @@ end
     io = IOContext(IOBuffer(), :displaysize=>(10,20), :limit=>true)
     show(io, df_wide, splitcols=false, allcols=false)
     str = String(take!(io.io))
-    @test str == """1×3 DataFrame. Omitted printing of 2 columns
-                 │ Row │ x1    │
-                 │     │ Int64 │
-                 ├─────┼───────┤
-                 │ 1   │ 1     │"""
+    @test str == """
```

Comment:
It would be ideal if we had tests for all cases of combinations of `allcols` and `splitcols` and overwide column in different placements you implement (maybe we have it already, but I am just asking to be sure).